src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -20,6 +20,9 @@
   </li>
 
   <li>
+    <i ng-click="zoomIn()" class="time-nav-btn fa fa-search-plus ng-scope"></i>
```

Comment:
minor: I wouldn't mind tooltips on these buttons actually. 'move [forward|backward] in time', 'zoom out timeline', 'zoom in timeline'